scripts/release/sync-homebrew-cask.sh

```
@@ -50,20 +50,23 @@ if [[ ! -f "$DMG_PATH" ]]; then
   exit 1
 fi
 
-if [[ -z "$APP_VERSION" || "$APP_VERSION" == '$('* || -z "$APP_BUILD" || "$APP_BUILD" == '$('* ]]; then
+if [[ -z "$APP_VERSION" || "$APP_VERSION" == '$('* ]]; then
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)